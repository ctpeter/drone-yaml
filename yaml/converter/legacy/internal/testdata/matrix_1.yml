
pipeline:
  test:
    image: golang:${GO_VERSION}
    commands:
    - go test -v ./...

get_env_var_from_vault_sg_staging:
  redis:
    image: redis:2.6

matrix:
  include:
    - GO_VERSION: 1.11
      REDIS_VERSION: 2.6
    - GO_VERSION: 1.10
      REDIS_VERSION: 2.6
    - GO_VERSION: 1.9
      REDIS_VERSION: 2.6
