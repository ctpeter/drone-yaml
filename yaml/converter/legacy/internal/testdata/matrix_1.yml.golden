---
kind: pipeline
name: matrix-1

platform:
  os: linux
  arch: amd64

steps:
- name: test
  pull: if-not-exists
  image: golang:1.11
  commands:
  - go test -v ./...

get_env_var_from_vault_sg_staging:
- name: redis
  pull: if-not-exists
  image: redis:2.6

---
kind: pipeline
name: matrix-2

platform:
  os: linux
  arch: amd64

steps:
- name: test
  pull: if-not-exists
  image: golang:1.10
  commands:
  - go test -v ./...

get_env_var_from_vault_sg_staging:
- name: redis
  pull: if-not-exists
  image: redis:2.6

---
kind: pipeline
name: matrix-3

platform:
  os: linux
  arch: amd64

steps:
- name: test
  pull: if-not-exists
  image: golang:1.9
  commands:
  - go test -v ./...

get_env_var_from_vault_sg_staging:
- name: redis
  pull: if-not-exists
  image: redis:2.6

...
